SwiftJym/Assets.xcassets/Bg.colorset/Contents.json

```diff
@@ -23,9 +23,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0.969",
-          "green" : "0.949",
-          "red" : "0.949"
+          "blue" : "0.039",
+          "green" : "0.039",
+          "red" : "0.039"
```

Comment:
I've picked a slightly lighter black (someting like what u did with the light scheme) but can be changed obviously

Reply:
Good morning to you my friend. The colour works great, thank you.